test/utils/assertions.js

```diff
@@ -1,11 +1,45 @@
 import _ from 'lodash';
 import path from 'path';
 import should from 'should';
-should.Assertion.add('equalFileSystemPath', function (value, description) {
+import fs from 'fs/promises';
+
+should.Assertion.add('equalFileSystemPath', function(value, description) {
 	value = path.normalize(value);
-	if (process.platform == 'win32' && _.startsWith(value, path.sep)) {
+	if (process.platform === 'win32' && _.startsWith(value, path.sep)) {
```

Comment:
Could you please also replace `_.startsWith` with native String's startsWith and get rid of lodash here?